tests/conftest.py

```diff
@@ -384,11 +384,12 @@ def _post_collection(th_client, project, oauth_key, oauth_secret,
                                                 data=jsondata, oauth_key=oauth_key,
                                                 oauth_secret=oauth_secret,
                                                 method='POST')
-        getattr(TestApp(application), 'post')(
+        resp = getattr(TestApp(application), 'post')(
```

Comment:
added this so we could test if it returns a 301 vs. a 200 when posting to the Objectstore.